LEAF_Request_Portal/sources/Form.php

```diff
@@ -3628,7 +3636,7 @@ public function query($inQuery)
                 $indicatorIDs .= $indicatorID . ',';
             }
 
-            return $this->getCustomData($data, $indicatorIDs);
+            return $this->getCustomData($data, $indicatorIDs, $alreadyCheckedReadAccess);
```

Comment:
Functions should have only one return.  Can this be turned to a variable and then passed back with the final return?

Reply:
Updated.